test/e2e/flask/btc/create-btc-account.spec.ts

```diff
@@ -91,8 +90,14 @@ describe('Create BTC Account', function (this: Suite) {
 
         // Recreate account and check that the address is the same
         await headerNavbar.openAccountMenu();
-        await accountListPage.check_pageIsLoaded();
```

Comment:
is there a reason we removed `await accountListPage.check_pageIsLoaded();` ?

Reply:
reverted